atc/worker/volume.go

```diff
@@ -26,6 +26,7 @@ type Volume interface {
 	COWStrategy() baggageclaim.COWStrategy
 
 	InitializeResourceCache(db.UsedResourceCache) error
+	GetResourceCacheId() int
```

Comment:
was there a reason to make it `Id` instead of `ID`? I noticed that there's another `GetResourceCacheID` in this PR, but this one had that `d` lowercased 

```suggestion
	GetResourceCacheID() int
```

Reply:
fixed